cogfiles/birthdays.py

```diff
@@ -21,23 +22,36 @@ class Month(Enum):
     November = 11
     December = 12
 
+@dataclass(frozen=True, order=True)
+class Birthday(JsonSerializable):
+    user: discord.Member
+    date: datetime.date = field(compare=False)

```

Comment:
Wait a sec, why is the `date` property in `Birthday` set to `compare=False`? If the issue is that `datetime` is unhashable, we should set `hash=False` on the property instead, so we can still compare Birthday objects by their date naturally.